xmodule/video_block/video_block.py

```diff
@@ -541,6 +542,13 @@ def is_public_sharing_enabled(self):
         else:
             return self.public_access
 
+    def is_transcript_feedback_enabled(self):
+        """
+        Is transcript feedback enabled for this video?
+        """
+        # TODO
+        return True
```

Comment:
Should this default to false?

Reply:
For the moment I'm just sending True to be able to work with it, but in the last ticket we will get the necessary data to know if we want to show it or not and also default it to False.